src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -1023,4 +1043,161 @@ private String getMagicPermission(String endpoint) {
     return String.format("%s.execute", Base64.encode(endpoint));
   }
 
+  @Test
+  public void testStoreSaveRefreshToken(TestContext context) {
+    var ts = new RefreshTokenStore(vertx, tenant);
+    var rt = new RefreshToken(tenant, "jones", userUUID, "http://localhost:" + port);
+    ts.saveToken(rt)
+      .compose(x -> ts.checkTokenNotRevoked(rt))
+      .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testStoreRefreshTokenNotFound(TestContext context) {
+    var ts = new RefreshTokenStore(vertx, tenant);
+    // A RefreshToken which doesn't exist is treated as revoked.
+    var unsavedToken = new RefreshToken(tenant, "jones", userUUID, "http://localhost:" + port);
+    ts.checkTokenNotRevoked(unsavedToken).onComplete(context.asyncAssertFailure());
+  }
+
+  @Test
+  public void testStoreSaveApiToken(TestContext context) {
+    var ts = new ApiTokenStore(vertx, tenant, tokenCreator);
+    var apiToken = new ApiToken(tenant);
+    ts.saveToken(apiToken).compose(x ->  ts.checkTokenNotRevoked(apiToken))
+      .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testStoreApiTokenNotFound(TestContext context) {
+    var ts = new ApiTokenStore(vertx, tenant, tokenCreator);
+    // A ApiToken which doesn't exist in storage is treated as revoked.
```

Comment:
This comment is missing from checkTokenNotRevoked's javadoc.
The error message should reflect that: "The token doesn't exist or has been revoked."

Reply:
Added the error message to `getRow` and also now check for it in the test.